urdf_model/include/urdf_model/pose.h

```diff
@@ -164,6 +201,18 @@ class Rotation
     setFromRPY(rpy.x, rpy.y, rpy.z);
   }
 
+  void initQuaternions(const std::string &rotation_str)
```

Comment:
This is really `initQuaternion`, right?  That is, it will only allow one set of "x y z w".

Reply:
Ah yes, only one quaternion can be initialized. I have rename the function `initQuaternion` in both pull requests.